crates/btc-relay/src/lib.rs

```diff
@@ -317,6 +317,13 @@ decl_module! {
             Self::clear_block_error(block_hash, error)
         }
 
+        #[weight = 0]
+        #[transactional]
+        fn set_parachain_confirmations(origin, period: T::BlockNumber) {
```

Comment:
This can be removed when Greg the `system.set_storage(key, value)` to the clients